test/parallel/test-zlib-from-concatenated-gzip.js

```diff
@@ -20,13 +23,13 @@ assert.strictEqual(zlib.gunzipSync(data).toString(), 'abcdef');
 zlib.gunzip(data, common.mustCall((err, result) => {
   assert.ifError(err);
   assert.strictEqual(result.toString(), 'abcdef',
-                     'result should match original string');
+                     'result should match original string: ' + (abc + def));
```

Comment:
The more important part is actually what `result.toString()` looks like. Please change it to something like
```js
`${result.toString()} should equal ${abc + def}`
```